README.md

```diff
@@ -1,3 +1,233 @@
 # preprocessing-moma
 
 **preprocessing-moma** is an Enduro preprocessing workflow for MoMA SIPs.
+It removes unwanted files/directories from the SIP based on configuration.
+
+- [Configuration](#configuration)
+- [Local environment](#local-environment)
+- [Makefile](#makefile)
+
+## Configuration
+
+The preprocessing workers need to share the filesystem with Enduro's a3m or
+Archivematica workers. They must be connected to the same Temporal server
+and related to each other with the namespace, task queue and workflow name.
+
+### Preprocessing
+
+The entire configuration for the preprocessing worker:
+
+```toml
+debug = false
+verbosity = 0
+sharedPath = "/home/enduro/preprocessing"
+
+[temporal]
+address = "temporal.enduro-sdps:7233"
+namespace = "default"
+taskQueue = "preprocessing"
+workflowName = "preprocessing"
+
+[worker]
+maxConcurrentSessions = 1
+
+[removeFiles]
+removeNames = ".DS_Store"
+```
+
+### Enduro
+
+The preprocessing section for Enduro's configuration:
+
+```toml
+[preprocessing]
+enabled = true
+extract = false
+sharedPath = "/home/enduro/preprocessing"
+
+[preprocessing.temporal]
+namespace = "default"
+taskQueue = "preprocessing"
+workflowName = "preprocessing"
+```
+
+## Local environment
+
+### Requirements
+
+This project uses Tilt to set up a local environment building the Docker images
+in a Kubernetes cluster. It has been tested with k3d, Minikube and Kind.
+
+- [Docker] (v18.09+)
+- [kubectl]
+- [Tilt] (v0.22.2+)
+
+A local Kubernetes cluster:
+
+- [k3d] _(recomended, used in CI)_
+- [Minikube] _(tested)_
+- [Kind] _(tested)_
+
+It can run with other solutions like Microk8s or Docker for Desktop/Mac and
+even against remote clusters, check Tilt's [Choosing a Local Dev Cluster] and
+[Install] documentation for more information to install these requirements.
+
+Additionally, follow the [Manage Docker as a non-root user] post-install guide
+so that you don’t have to run Tilt with `sudo`. _Note that managing Docker as a
+non-root user is **different** from running the docker daemon as a non-root user
+(rootless)._
+
+### Requirements for development
+
+While we run the services inside a Kubernetes cluster we recomend to install
```

Comment:
Edit: "we recommend ~to install~installing Go and other tools locally" (emphasis